src/main/resources/templates/assign-enquiry-notification.json

```diff
@@ -3,7 +3,7 @@
 		"de": "assign-enquiry-notification.html"
 	},
 	"subject": {
-		"de": "Neue_r Ratsuchende_r"
+		"key": "mail.subject.assign.enquiry.notification"
```

Comment:
for me 
direct-enquiry-notification-consultant.json
"de": "Neue_r Ratsuchende_r: Neue Erstanfrage über Kontaktlink [PLZ: ${plz}]"

and 
reassign-request-notification.json
"de": "Anfrage Neuzuweisung"

are missing.

Reply:
ok, the direct-enquiry-notification-consultant.json is missing in the mail-service... nice.
The question is if we should sync that

Reply:
Added key for the template but not the template itself.

Because the templates are not used in the Maiil service and are overwritten both in the Kubernetes repro and in the Config repos (SaaS, Tsys, etc.). The result does not make the current situation any clearer, but that was the least of the effort.